### PR TITLE
Added a on_message for twitter scammers

### PR DESCRIPTION
**Pull request by Itz-Cook1e**
Added support for users who ask if Mark has a Twitter and are being scammed by a fake Mark.

### CODE REVIEW

**Review by Itz-Cook1e — COMMENTED** [review]
Made a mistake on the message.channel.send.

File: cogs/moderation.py [hunk 1]
```diff
@@ -86,6 +86,11 @@ async def report(self, ctx, user: discord.Member, *reason):
                 
                 print(error)
                 return await ctx.send(embed=discord.Embed(title="Error!", description="Error making the report! Please try again."))
+            
+    @commands.Cog.listener(name='on_message')
+    async def mark_twitter(self, message):
```
Comment (alfiephillips) [inline cogs/moderation.py:91]: Are you able to add checks for a twitter link?

File: cogs/moderation.py [hunk 2]
```diff
@@ -86,6 +86,11 @@ async def report(self, ctx, user: discord.Member, *reason):
                 
                 print(error)
                 return await ctx.send(embed=discord.Embed(title="Error!", description="Error making the report! Please try again."))
+            
+    @commands.Cog.listener(name='on_message')
```
Comment (alfiephillips) [inline cogs/moderation.py:90]: Can you add some extra documentation under the function, just cleans up the code a bit.

**Review by alfiephillips — CHANGES_REQUESTED** [review]
Please review the comments i have left for you.

**Review by Itz-Cook1e — COMMENTED** [review]
Added documentation.

File: cogs/moderation.py (hunk 2 above)
Comment (Itz-Cook1e) [reply cogs/moderation.py:90]: Submitted changes.

File: cogs/moderation.py (hunk 1 above)
Comment (Itz-Cook1e) [reply cogs/moderation.py:91]: Explained this on Discord.

**Pull request merged**